docs/user_guide.md

```diff
@@ -174,32 +201,50 @@ $ docker push 10.117.169.182/demo/ubuntu:14.04
 the repository is no longer managed in Harbor, however, the files of the repository still remain in Harbor's storage.  
 
-![browse project](img/new_delete_repository.png)
+![browse project](img/new_delete_repo.png)
+![browse project](img/new_delete_tag.png)
 
-**CAUTION: If both tag A and tag B refer to the same image, after deleting tag A, B will also get deleted.**  
+**CAUTION: If both tag A and tag B refer to the same image, after deleting tag A, B will also get deleted. if you enabled content trust, you need to use notary command line tool to delete the tag's signature before you delete an image.**  
 
 Next, delete the actual files of the repository using the registry's garbage collection(GC). Make sure that no one is pushing images or Harbor is not running at all before you perform a GC. If someone were pushing an image while GC is running, there is a risk that the image's layers will be mistakenly deleted which results in a corrupted image. So before running GC, a preferred approach is to stop Harbor first.  
 
 Run the below commands on the host which Harbor is deployed on to preview what files/images will be affected: 
 
 ```sh
 $ docker-compose stop
-$ docker run -it --name gc --rm --volumes-from registry registry:2.6.0 garbage-collect --dry-run /etc/registry/config.yml
+
+$ docker run -it --name gc --rm --volumes-from registry vmware/registry:photon-2.6.0 garbage-collect --dry-run /etc/registry/config.yml
+
 ```  
 **NOTE:** The above option "--dry-run" will print the progress without removing any data.  
 
 Verify the result of the above test, then use the below commands to perform garbage collection and restart Harbor. 
 
 ```sh
-$ docker run -it --name gc --rm --volumes-from registry registry:2.6.0 garbage-collect  /etc/registry/config.yml
+
+$ docker run -it --name gc --rm --volumes-from registry vmware/registry:photon-2.6.0 garbage-collect  /etc/registry/config.yml
+
 $ docker-compose start
 ```  
 
 For more information about GC, please see [GC](https://github.com/docker/docker.github.io/blob/master/registry/garbage-collection.md).  
+
+### Content trust  
+If you want to enable content trust to ensure that images are signed, set two environment variables in the terminal before push or pull any image:
+```sh
+export DOCKER_CONTENT_TRUST=1
+export DOCKER_CONTENT_TRUST_SERVER=https://10.117.169.182:4443
+```
+If you are using a self-signed cert, make sure to copy the CA cert into ```/etc/docker/certs.d/10.117.169.182``` and ```$HOME/.docker/tls/10.117.169.182:4443/```. when an images is signed, it is shown in the Web UI.  
+**Note: Replace "10.117.169.182" with the IP address or domain name of your Harbor node. In order to use content trust,HTTPS must be enabled in Harbor.**  
+  
+
+When an image is signed, image will have a tick show in UI, unsigned will have a cross show in UI.  

```

Comment:
When an image is signed, it has a tick shown in UI; otherwise, a cross is displayed instead.